convert2rhel/cert.py

```diff
@@ -58,3 +60,33 @@ def install(self):
             loggerinst.critical("OSError({0}): {1}".format(err.errno, err.strerror))
 
         loggerinst.debug("Certificate copied to %s." % self._system_cert_dir)
+
+    def remove(self):
+        """Removes certificate (.pem), which was copied in install method"""
+        loggerinst.info("Removing copy of certificate from %s." % self._system_cert_dir)
+
+        try:
+            os.remove(os.path.join(self._system_cert_dir, self._get_cert_filename()))
+        except OSError as err:
+            loggerinst.error("OSError({0}): {1}".format(err.errno, err.strerror))
+
+        loggerinst.debug("Certificate removed")
+
+    @staticmethod
+    def _get_cert_filename():
+        """ Same as _get_cert_path, but returns just filename, not whole path """
+        loggerinst = logging.getLogger(__name__)
+        cert_dir = os.path.join(utils.DATA_DIR, "rhel-certs")
+        if not os.access(cert_dir, os.R_OK | os.W_OK):
+            loggerinst.critical("Error: Could not access %s." % cert_dir)
+        pem_filename = None
+        for filename in os.listdir(cert_dir):
+            if filename.endswith(".pem"):
+                pem_filename = filename
+                break
+        if not pem_filename:
+            loggerinst.critical(
+                "Error: System certificate (.pem) not found in %s." % cert_dir
+            )
+
+        return pem_filename
```

Comment:
Since this is the exact same implementation as `_get_cert_path` as you mentioned, I'd recommend to consolidate these two into one function

Something like `_get_cert()` that returns a tuple of name and path `return pem_filename, os.path.join(cert_dir, pem_filename)`. Then set these two as class variables in the `__init__`